config/tablet/default.yaml

```diff
@@ -99,7 +99,6 @@ hotRowProtection:
 consolidator: enable|disable|notOnPrimary # enable-consolidator, enable-consolidator-replicas
 passthroughDML: false                    # queryserver-config-passthrough-dmls
 streamBufferSize: 32768                  # queryserver-config-stream-buffer-size
-queryCacheSize: 5000                     # queryserver-config-query-cache-size
```

Comment:
This flag was removed a while ago, I took the liberty of removing this from the config file.